linters/ansible-lint/plugin.yaml

```diff
@@ -1,4 +1,11 @@
 version: 0.1
+tools:
+  definitions:
+    - name: ansible-lint
+      runtime: python
+      package: ansible-lint
+      extra_packages: [ansible]
+      shims: [ansible-lint]
```

Comment:
Can we add ansible as a shim too (or make it another tool that's in the dependency)?

Reply:
Related, how's the ansible/ansible-lint experience?